packages/cli/src/Generate.ts

```diff
@@ -154,6 +155,7 @@ ${bold('Examples')}
           Boolean(process.env.PRISMA_GENERATE_DATAPROXY) || // legacy, keep for backwards compatibility
           Boolean(process.env.PRISMA_GENERATE_ACCELERATE) || // legacy, keep for backwards compatibility
           Boolean(process.env.PRISMA_GENERATE_NO_ENGINE),
+        noHints: Boolean(args['--no-hints']),
```

Comment:
Why forward this to generators? At no point this value actually get used right now.

Reply:
It's gone now, thanks for catching this.